app/models/usergroup.rb

```diff
@@ -112,4 +116,12 @@ def ensure_last_admin_group_is_not_deleted
   def other_admins
     User.unscoped.only_admin.except_hidden - all_users
   end
+
+  def ssh_keys
+    users.flat_map(&:ssh_keys)
```

Comment:
`all_users` instead, otherwise it will not return the SSH keys for nested user groups